specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -330,7 +344,7 @@ def verify_kzg_proof_multi_impl(commitment: KZGCommitment,
 #### `coset_for_cell`
 
 ```python
-def coset_for_cell(cell_id: int) -> Cell:
+def coset_for_cell(cell_id: CellID) -> Cell:
```

Comment:
Hmm, I think `cell_id` should be `column_index` here.

Reply:
In practice, `column_id` and `cell_id` can be used interchangeably in this function, since each cell corresponds to a column in a one-to-one fashion.

I think it's better for us to choose which notion is more intuitive in a separate PR.